examples/fastapi/arcade_example_fastapi/main.py

```diff
@@ -33,14 +33,13 @@ async def postChat(request: ChatRequest, tool_choice: str = "execute"):
             model="gpt-4o-mini",
             max_tokens=500,
             tools=[
-                # "Google.GetEmails",
-                # "Google.SearchEmailsByHeader",
-                # "Google.WriteDraft",
-                # "GitHub.CountStargazers",
-                # "GitHub.SetStarred",
-                # "GitHub.SearchIssues",
-                # "Slack.SendDmToUser",
-                # "Slack.SendMessageToChannel",
+                "Math.Add",
+                "Math.Subtract",
+                "Math.Multiply",
+                "Math.Divide",
+                "Math.Sqrt",
+                # Other tools can be added as needed:
+                # "Math.SumList"
```

Comment:
This example shows adding the math toolkit, so `tools` should reference math tools. Made this clearer!